## Detect variations in [RNA-seq](https://rnabio.org) with [nextflow](https://www.nextflow.io/docs/latest/install.html)

```
git clone https://github.com/animesh/rnaseq_nextflow
cd rnaseq_nextflow/
awk -F ',' '{print $1}' samples.csv | sed 's/Rep[123]//g' | sed 's/sample/patient/g' > patient.txt
paste -d "," patient.txt samples.csv > samplesheet.csv
cut -d',' -f1-4 samplesheet.csv > samples.csv
./nextflow self-update    
./nextflow run https://github.com/animesh/sarek --input samples.csv --outdir results -profile docker --tools  haplotypecaller,vep   -resume
```

## Results

After running the pipeline, results will be available in the `results` directory. This directory will contain:

- [Variant call files (VCF) generated by HaplotypeCaller](./results/variant_calling/haplotypecaller)
- [Variant annotation files produced by VEP](./results/reports/EnsemblVEP/haplotypecaller)
- [Quality control metrics](./results/multiqc/multiqc_report.html)

can explore the `results` directory to find output files for downstream analysis or visualization.

